compiler/passes/codegen.cpp

```diff
@@ -813,7 +828,7 @@ static void codegen_header() {
   // identifiers.  This can be disabled via the --munge-user-idents
   // flag.
   //
-  if (fMungeUserIdents) {
+  if (fMungeUserIdents && isHeader) {
```

Comment:
I'm guessing the isHeader check here is because otherwise the name munging would occur twice?